src/main/java/com/assignment1/item/ItemEntity.java

```diff
@@ -0,0 +1,94 @@
+package com.assignment1.item;
+
+import com.assignment1.Type;
+
+/** Class as blueprint for Item Entity. */
+@SuppressWarnings("PMD.DataClass")
```

Comment:
this PMD can be solved. if you design the interfaces properly this PMD could have been avoided

Reply:
Done this. PMD is resolved.